src/prefect/server/services/scheduler.py

```diff
@@ -302,8 +301,7 @@ def _get_select_deployments_to_schedule_query(self, db: PrefectDBInterface):
         query = (
             sa.select(db.Deployment.id)
             .where(
-                db.Deployment.is_schedule_active.is_(True),
-                db.Deployment.schedule.is_not(None),
+                db.Deployment.paused.is_not(True),
```

Comment:
kinda the same q as above but are we just removing `.is_schedule_active` and `.schedule`? I'm confused why they are not relevant anymore

Reply:
We're adding deployment_schedules, and now each schedule can be active independently

Reply:
I do get that but I guess what i'm really asking is why is it no longer important that a user may not have added a `schedule`, i.e.  `db.Deployment.schedule.is_not(None)`? Is it because we'll no longer be populating it there?

Reply:
er, why do we no longer care that they have not added a schedule

Reply:
We'll be removing `Deployment.schedule` and `Deployment.schedules` will be the canonical source of truth.

Reply:
thank you, that's what I was trying to figure out! cool, i must have missed that PR

Reply:
No worries! It's kind of a weird state at the moment, Nebula still has both and it's doing some syncing and the like between the tables. But we want to drop a 'fully-formed' version into OSS so that we don't have the possibility of that syncing to fail between releases. So the implementation here is a bit different from nebula.

Reply:
gotcha, thanks for the explanation :)

Reply:
A much better explanation than my attempt!